Q4.txt

```diff
@@ -0,0 +1,28 @@
+JavaScript:
+
+var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var x=0;
+var isprint=true;
+while (x<a.length){
```

Comment:
@aqattan so, we need to print every other element in reverse. That is all. So,
john
ekim
susan
ttam
etc.